core/app/[locale]/(default)/product/[slug]/page.tsx

```diff
@@ -327,6 +327,7 @@ export default async function Product({ params: { locale, slug }, searchParams }
       <ProductDetail action={addToCart} fields={formattedFields} product={formattedProduct} />
 
       <ProductDescription
+        product={formattedProduct}
```

Comment:
Passing product so we can properly scope the snapshot/label the changes for the Publish dialog.